src/toil/lib/bioio.py

```
@@ -58,6 +58,18 @@ def addLoggingFileHandler(fileName, rotatingLogging=False):
     rootLogger.addHandler(handler)
     return handler
 
+def addSyslogHandler(destination):
+    from logging.handlers import SysLogHandler
+
+    if ':' in destination:
+        address, port = destination.split(':')
+        address = address or "localhost"
+        port = int(port or '514')
```

This is handling the case of --syslogDest 'localhost:' which should be an error anyway?

Also, if destination doesn't have ':' in it, wont this handler fail instead of assuming a default port?

i.e. --syslogDest localhost  

Should work gracefully.